docs/get-started/lotus/chain.md

```diff
@@ -17,113 +15,163 @@ Lotus will automatically sync to the latest _chain head_ by:
+:::warning
+These lightweight state snapshots **do not contain any message receipts**. To get message receipts, you need to sync your Lotus node from the genesis block without using any of these snapshots.
+:::
+
+1.  Download the most recent lightweight snapshot:
+
+    ```bash
+    wget https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car
+    ```
 
-# An alternative is to download first and use the file
-lotus daemon --import-snapshot <filename.car>
+1.  Check the sha256sum of the download:
 
-# The sha256sum is stored alongside the interim snapshot and can be obtained via
-curl -sI https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car \
-| perl -ne '/^x-amz-website-redirect-location:(.+)\.car\s*$/ && print "$1.sha256sum"' \
-| xargs curl -s
+    ```bash
+    curl -sI https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car | perl -ne '/^x-amz-website-redirect-location:(.+)\.car\s\*$/ && print "$1.sha256sum"' | xargs curl -s
+    ```
+
+1.  Start the Lotus daemon using `--import-snapshot`:
+
+    ```bash
+    lotus daemon --import-snapshot minimal_finality_stateroots_latest.car
+    ```
+
+We strongly recommend you verify the checksum of the download. However, you can skip the `sha256sum` check and use the snapshot URL directly if you'd prefer:
+
+```bash
+lotus daemon --import-snapshot https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car
 ```
 
-::: warning
-If you do not trust the source of the snapshot or simply want Lotus to validate the chain, you need to follow the steps in the next section. Complete validation of the chain is an order of magnitude slower, and is expected to take multiple days.
-:::
+#### Get message receipts
+
+The lightweight snapshot does not contain any message receipts. However, you can get them retroactively once you have imported the lightweight snapshot:
+
+1. Import the lightweight snapshot [as normal](#lightweight-snapshot).
+1. Create a new environment variable called `LOTUS_ENABLE_CHAINSTORE_FALLBACK` and set it to `1`:
 
-### Syncing from a full snapshot (mainnet)
+   ```profile
+   $LOTUS_ENABLE_CHAINSTORE_FALLBACK=1
+   ```
 
-An alternative to trusted state snapshots are full chain snapshots ([available here](https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/complete_chain_with_finality_stateroots_latest.car)), which are significantly bigger and therefore slower to download, but will contain every block from genesis until the current tipset. In addition to the trusted import option described above, these complete snapshots can be trustlessly imported by fully recalculating the state during import. Simply supply the `--import-chain` option instead:
+1. <!-- USE LOTUS SET-HEAD TO DO SOMETHING -->
```

Comment:
@ribasushi @jennijuju could either of your expand on the actual steps needed here? I'm completely guessing, and don't have a miner/node available to test.